RPG-game-Hung/classes/actions.py

```diff
@@ -0,0 +1,27 @@
+from termcolor import colored
+
+from classes.colors import Color
+
+
+class Action:
+    text = None  # For printing during player's action selection
+    collection = {
+        1: ["Slap", "target.slapped()"],
+        2: ["Magic", "Magic.enchant(attacker,target)"],
+        3: ["Item", "Item.use(attacker,target)"],
+    }
+
```

Comment:
this `dict` might be over-complicated, as a reader I actually not sure what's that for. I would assume that you will call the methods when choosing the actions, however, this is unnecessary as it will be harder to debug, and might cause more bugs in situation such as when `Item` or `Magic` class is modify, and `enchant` and `use` are no longer a `classmethod`, this will break, Or if the `use` and `enchant` method is modified, and it causes you more work to change it here as well if you remember it.